client/src/nv_ingest_client/client/interface.py

```diff
@@ -392,10 +392,15 @@ def extract(self, **kwargs: Any) -> "Ingestor":
         """
         extract_tables = kwargs.pop("extract_tables", True)
         extract_charts = kwargs.pop("extract_charts", True)
+        extract_infographics = kwargs.pop("extract_infographics", True)
```

Comment:
Open question: `extract_infographics` is set to True by default, but a recent test shows that there may be a performance hit with infographics (~3 pages/sec drop from ~12 pages/sec to ~9 pages/sec). One option is we could make the default `extract_infographics=False` to avoid the throughput drop in the default path, and let the user explicitly enable the inforgraphics mode if they want it.

Reply:
@randerzander This seems like it should be opt-in functionality with the given perf hit. Thoughts?

Reply:
I went ahead and made it an opt-in in [6413b8d](https://github.com/NVIDIA/nv-ingest/pull/483/commits/6413b8d4663a650f32534b3b81e9099a544e6cdf). `extract_infographics` is False by default and users will have to explicitly turn it on for now.